org.blender.Blender.json

```diff
@@ -202,6 +202,30 @@
                 "/share"
             ]
         },
+        {
+            "name": "opensubdiv",
+            "buildsystem": "cmake-ninja",
+            "builddir": true,
+            "config-opts": [
```

Comment:
Could you add (as first option) the `"-DCMAKE_BUILD_TYPE=RelWithDebInfo" option like the other CMake modules?